scenarios/test/various_validator_versions.yml

```
@@ -0,0 +1,44 @@
+# This scenario simulates a network with the various versions of validators.
+# It stars a few startup validator nodes, each running a different version of the client.
+# It deploys a few applications and sends a decent number of transactions.
+
+# The name of the scenario
+name: Various Validator Versions
+
+# The duration of the scenario's runtime, in seconds.
+duration: 180
+
+# Initial validator nodes in the network.
+validators:
+  - name: validator-1
+  - name: validator-2
+    imagename: "sonic:v2.0.2"
+  - name: validator-3
+    instances: 2
+    imagename: "sonic:v2.0.1"
+  - name: validator-4
+    instances: 3
+    imagename: "sonic"
```

Since it is the demo for how to use this option, I would suggest to rename the validators to produce nice instance names:
```suggestion
  - name: validator-dev
  - name: validator-v2.0.2
    imagename: "sonic:v2.0.2"
  - name: validator-v2.0.1
    instances: 2
    imagename: "sonic:v2.0.1"
  - name: validator-latest
    instances: 3
    imagename: "sonic"
```